src/main/java/ua/danit/final_project/entities/Schedule.java

```diff
@@ -40,4 +37,8 @@
   @EqualsAndHashCode.Exclude
   @Column(name = "end", nullable = false)
   private Time end;
+
+  private String uuid;
+
+  private Date expired;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "expired" private field. ([squid:S1068](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=108&issues=AWXEVMDG_eEZPtUmhoyt&open=AWXEVMDG_eEZPtUmhoyt)

[ISSUE_KEY]: AWXEVMDG_eEZPtUmhoyt